src/pmpo_c.h

```diff
@@ -40,9 +40,9 @@ void polympo_setMeshGeomTypePlanar_f(MPMesh_ptr p_mpmesh);
 void polympo_setMeshGeomTypeSpherical_f(MPMesh_ptr p_mpmesh);
 void polympo_setMeshSphereRadius_f(MPMesh_ptr p_mpmesh, const double sphereRadius);
 void polympo_setMeshNumVtxs_f(MPMesh_ptr p_mpmesh, const int numVtxs);
-int polympo_getMeshNumVtxs_f(MPMesh_ptr p_mpmesh);
+void polympo_getMeshNumVertices_f(MPMesh_ptr p_mpmesh, int & numVtxs);
 void polympo_setMeshNumElms_f(MPMesh_ptr p_mpmesh, const int numElms);
-int polympo_getMeshNumElms_f(MPMesh_ptr p_mpmesh);
```

Comment:
Are we also going to change the set functions names? @onkarsahni

Reply:
Use polympo_getMeshNumVtxs_f and polympo_getMeshNumElms_f